"Big" Update!

Danix421 opened the pull request:
Hi zyenapz, i'm Danix, a Python developer like you! Sorry if my english is bad but i only speak spanish and somthing of english. Exploring itch.io, i viewed your game and it's so cool, and i wanted to contribute. I'm now working in a "big" update for Caffeine Fueled, so this will be the changes:

1. The game title menu will be modified, so.. the zyenapz info will be erased from the title menu for be more "friendless" with the player.
2. The credits option will be added to the tile menu, and your name (and obvious, my name) will be added to it.
3. The save system will be totally changed, why if you erase the save file for restart the game, it will not run (and it's really, really, annoying).
4. The readme.txt file will be (and it's) updated with the last update info, my name in the credits, and the requeriments (pygame).
5. The source code (src) is the version that will be updated, the windows build will not uploaded, that you will have to do it B).
6. If i can, i will fix other bugs.

This is a draft pull request for one reason: the update it's not ready, but will be ready sometime.
Soo, if you wan't this update, please comment it to this draft to know it!
Well, i hope you like it!

Commit:
Added requeriments and updated the info

zyenapz commented:
Hi, Danix!

Thank you for playing my game and I'm glad you enjoyed it so much that you are willing to contribute! I have no other plans for Caffeine Fueled in the near future since I made it only for a game jam but you are welcome to contribute!

Here is my feedback for your contribution:

1. You have my permission to remove the huge developer logo. I admit it is not very good placing it there and that I should probably add it somewhere in the backdrop.
2. That's alright but don't forget to credit me as the original developer and display my name preferably in a prominent manner :). I prefer not using another menu for that, though. I prefer to just have my logo (and your logo, if you so wish) somewhere in the background in a subtle manner.
3. Yep, that's one of the 'shortcuts' I took because I was running out of time to submit it to the game jam. My subsequent game 'Low Key' implements a proper saving method that doesn't crash the game when you delete the save file. You can probably copy that 'saving' code from there. It's in my repository[zyenapz/Low-Key](https://github.com/zyenapz/Low-Key)
4. Again, that is alright but please credit me as the original developer!
5. That's alright. Just notify me and I'll release the build!
6. Thank you, I appreciate that :D 

Well, that's all I will say for now. I'm looking forward to your contribution =) If you need anything, or want to clarify anything then just ask me!

Thank you for wanting to contribute again 🥂

fmtayag commented on readme.txt:
```diff
@@ -11,21 +8,30 @@ What is this game?
-Programming, art, sound effects by zyenapz
-Font: 'Press Start' downloaded from codeman38
-Music: 'HappyBeat' downloaded from YouTube and made by Electro Baptist
+Programming by Danix and zyenapz, art & sound effects by zyenapz.
```
Please credit me as the original developer here, thanks! =)

Danix421 replied:
Ok, i will do it!

fmtayag commented on readme.txt:
```diff
@@ -11,21 +8,30 @@ What is this game?
\ No newline at end of file
+- Bugfixing regarding it not running on some systems.
+Update 1.0.3
+- Title menu modified, credits option in title menu added and
```
All good to me!

Danix421 replied:
Thanks!

fmtayag commented on readme.txt:
```diff
@@ -11,21 +8,30 @@ What is this game?
+A, LEFT 			    - Move Left
+D, RIGHT 			    - Move Right
+
+Requeriments
```
Oops, a typo here! It should be 'Requirements' !

Danix421 replied:
I will fix it, thanks.

Commit:
Fixed  "Requeriments" typo and added requeriments typo

Commit:
Added more feactures to update and fixed "Requeriments" typo

Danix421 commented:
Thanks, zyenapz, I will _only view_ the [Low Key](https://github.com/zyenapz/Low-Key) repo for reference, again, thanks.

Commit:
Added "Contact" info

Commit:
Added more contact info

Commit:
Upgraded update version

Danix421 commented:
Well, i updated the readme again. I added contact info and i upgraded the version number 1.0.3 (the mine) to 1.1, why the save update is " really big"

Danix421 commented:
One moment.. why there in the source code game.py is a class named "GameData" that is not used? Don't confuse it with game_data, that is the save file, i saw "GameData", and is a class

zyenapz commented:
Hey, Danix, sorry for the abrupt changes but I think I'm going to be too busy to review the updates that you will make. I admit that I hastily made the decision to accept you as a contributor without considering my schedule for this week.

To put it simply, I'm studying MonoGame C# (among other things) for my semester break. As such I will be unable to attend to any updates you will make as I will be extremely busy.

Sorry again, Danix!

Danix421 commented:
Ok, i will code the update and when you're not busy you can view it, thanks for let me know it!
